build_tools/buildkite/scripts/wait_for_pipeline_success.py

```diff
@@ -81,17 +87,28 @@ def main(args):
     build = bk.create_build()
   build_number = get_build_number(build)
   url = bk.get_url_for_build(build_number)
-  print(f"Waiting on {linkify(url)}")
+  ansi_link = ansi_linkify(url)
+  markdown_link = f"[{args.pipeline}]({url})"
+  print(f"Waiting on {ansi_link}")
+  if args.annotate:
+    # Don't explicitly set the context, so it can be set with an environment
```

Comment:
Where will be the environment variable set?

Reply:
In the steps that call this, where I forgot to actually set it. Fixed, thank you :-)